packages/devtools_app/lib/main.dart

```diff
@@ -6,12 +6,18 @@ import 'package:flutter/material.dart';
 
 import 'src/app.dart';
 import 'src/config_specific/framework_initialize/framework_initialize.dart';
+import 'src/preferences.dart';
 
 void main() async {
+  final preferences = PreferencesController();
+  // Wait for preferences to load before rendering the app to avoid a flash of
```

Comment:
I was hoping this was the main reason the inspector light mode theme was broken but that was too optimistic.